docs/conf.py

```diff
@@ -38,11 +38,18 @@
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
     "djangodocs",
-    'sphinx.ext.extlinks',
+    "sphinx.ext.extlinks",
```

Comment:
This change is unrelated so we can revert it.

Reply:
The quotes were inconsistent in that list. Thought it would be better to make them all double quotes.

Reply:
I know but we shouldn't mix unrelated cleanups with bugfixes.